erizo/generateEclipseProject.sh

```diff
@@ -6,9 +6,9 @@ BIN_DIR="build"
 if [ -d $BIN_DIR ]; then
   cd $BIN_DIR
   # Set to Debug to be able to debug in Eclipse
-  cmake -G"Eclipse CDT4 - Unix Makefiles" -D CMAKE_BUILD_TYPE=Debug ../src
+  cmake -G"Eclipse CDT4 - Unix Makefiles" -D ERIZO_BUILD_TYPE=debug ../src
```

Comment:
a more complete solution has already been merged and it is conflicting with this one, so please remove these changes